modules/nw-multus-bond-cni-object.adoc

```diff
@@ -44,6 +44,10 @@ The following table describes the configuration parameters for the Bond CNI plug
 |`string`
 |Specifies the bonding policy. 
 
+|`xmitHashPolicy`
+|`string`
+|Specifies the transmit hash policy for load balancing across the aggregated interfaces. This parameter defaults to `layer2` and supports the following values: `layer2`, `layer2+3` and `layer3+4`.
```

Comment:
```suggestion
|Specifies the transmit hash policy for load balancing across the aggregated interfaces. This parameter defaults to `layer2` and supports the following values: `layer2`, `layer2+3`, and `layer3+4`.
```

Reply:
Are these all the values it supports? I just realized that in the config example you mention `layer3` too but it's not listed here.